hackathon/models.py

```diff
@@ -66,7 +72,6 @@ class Hackathon(models.Model):
     def __str__(self):
         return self.display_name
 
-
```

Comment:
As per PEP8 there should be two blank lines between classes, please add back the removed line.

Reply:
done